nuxeo-java-client/src/main/java/org/nuxeo/client/objects/StringEntity.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     jcarsique
+ *
+ */
+
+package org.nuxeo.client.objects;
+
+/**
+ * @since 3.8.1
```

Comment:
I will bump the _master_ branch to `3.9.0-SNAPSHOT` as it is an improvement, could you update the various since to `3.9.0`?